CMakeLists.txt

```diff
@@ -76,8 +76,12 @@ if(ENABLE_CPACK)
 endif()
 
 #--- target for Doxygen documentation ------------------------------------------
-if(EDM4HEP_DOCUMENTATION)
+if(BUILD_DOCS)
   include(cmake/EDM4HEPDoxygen.cmake)
+configure_file(${CMAKE_CURRENT_SOURCE_DIR}/doc/edm4hep_diagram.svg
```

Comment:
indentation for these lines is not correct

Reply:
Fixed